src/OutboundServer/api.yaml

```diff
@@ -574,7 +574,13 @@ components:
         merchantClassificationCode:
           type: string
           description: Up to 4 digits specifying the sender's merchant classification, if known and applicable.
-
+    transferPartyFrom:
```

Comment:
Why not add extensionList as an optional property on the transferParty object itself? I believe extensions may be present on both parties in some use cases.

Reply:
Changes made as suggested.